Octokit/Models/Request/UpdateTeam.cs

```diff
@@ -37,7 +37,7 @@ public UpdateTeam(string name)
         /// <summary>
         /// The permission that new repositories will be added to the team with when none is specified (default: Pull)
         /// </summary>
-        public Permission? Permission { get; set; }
+        public TeamPermission? Permission { get; set; }
```

Comment:
Initially, I did model a separate enum to allow push, pull, or admin permissions, but then when I tested using the admin value, I got an error response from the api, so I removed that enum and opted for using TeamPermission.cs with both create and update. I have also opened an issue [here](https://github.com/github/rest-api-description/issues/1952)

Reply:
:facepalm: :facepalm: :facepalm: Oh, nice catch! Would you mind linking to that issue in a brief code comment to explain why the actual use differs from what's documented? In the future I would completely forget and be confused about this again.

Reply:
@kfcampbell I can see how this would be really confusing in the future without a comment. I did add a comment to provide the context of why permissions are used the way they are. Let me know what you think.

Reply:
Looks good, thank you!